src/test/java/test/concordion/internal/listener/BreadcrumbRendererTest.java

```diff
@@ -47,25 +35,12 @@ public BreadcrumbRendererTest() {
     }
 
     @Test
-    public void logsNameOfErroneousHtmlFileOnParseError() {
```

Comment:
I'm keen that we keep the tests, since the functionality shouldn't change. Can we change to use http://stefanbirkner.github.io/system-rules/ instead of stubLogger?